test/util/TestEntity.js

```diff
@@ -0,0 +1,16 @@
+require('require-self-ref')
```

Comment:
Thoughts on putting this file in `test/dao/util/` to keep utils alongside their consuming tests? (assuming we adopt that sort of directory structure)